content/fr/guides/directory-structure/static.md

```diff
@@ -0,0 +1,90 @@
+---
+title: static
+description: Le répertoire `static` est directement relié à la racine du serveur () et contient des fichiers qui ne seront probablement pas modifiés. Tous les fichiers inclus sont automatiquement servis par Nuxt et accessibles via l'URL racine de votre projet.
+position: 12
+category: directory-structure
+csb_link: https://codesandbox.io/embed/github/nuxt-academy/guides-examples/tree/master/04_directory_structure/13_static?fontsize=14&hidenavigation=1&theme=dark
+questions:
```

Comment:
would be great to translate the quiz too as we will be launching this soon and if we don't translate it then you will have the english quiz on the page 🙁

Reply:
Haha, I was going to ask about that one ! 😁
Did it anyway because I was kinda expecting that you wished to have it translated too. 😊

Any ETA for the release of the guides? Gonna hurry myself! 😶

Reply:
We will release it as soon as it is all almost translated so yeah the faster it gets done the faster it gets released but hey no pressure 🤪